website/snippets/_config-description-resource.md

```diff
@@ -0,0 +1 @@
+Resource-specific configurations are applied to individual dbt resources. They influence specific attributes of the resource, such as [`sql-header`](/reference/resource-configs/sql_header), [`column_types`](/reference/resource-configs/column_types), [`strategy`](/reference/resource-configs/strategy), and so on. You can define these settings in the project file (`dbt_project.yml`), a property file (`models/properties.yml` for models, similarly for other resources), or within the resource’s file using the `{{ config() }}` macro.
```

Comment:
Since the code example underneath the snippet shows the _actual_ resource-specific configs (along with hyperlinks) supported by that particular resource type, what would you think about simplifying it to this?

```suggestion
Resource-specific configurations are applicable to only one dbt resource type rather than multiple resource types. You can define these settings in the project file (`dbt_project.yml`), a property file (`models/properties.yml` for models, similarly for other resources), or within the resource’s file using the `{{ config() }}` macro.
```